staging/src/k8s.io/client-go/util/prompt/prompt.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package prompt
```

Comment:
Why is this in client-go? This doesn't seem like the right place for this functionality.

Reply:
@lavalamp see https://github.com/kubernetes/kubernetes/pull/45260#issuecomment-300348949

Reply:
Ugh, I didn't realize this functionality was already in client-go. I don't think prompt functions are right to have in client-go (they should be either in a generic utility dumping ground or appear specifically where they are used). I guess I won't block this.

Reply:
Can you add a TODO that this package needs to move out of client-go?

Reply:
Agree. At most, the prompting loader should take an interface to actually  do the prompt, but I don't think the utility belongs in this repo